### PR TITLE
[stable/mediawiki] Use storageClassName for mediawiki.

### PR DESCRIPTION
**Pull request by gyliu513**
Fixed mediawiki for #1869

/cc @prydonius

### CODE REVIEW

File: stable/mediawiki/templates/apache-pvc.yaml [hunk 1]
```diff
@@ -8,16 +8,17 @@ metadata:
...
-  {{- else }}
-    volume.alpha.kubernetes.io/storage-class: default
-  {{- end }}
 spec:
   accessModes:
     - {{ .Values.persistence.apache.accessMode | quote }}
   resources:
     requests:
       storage: {{ .Values.persistence.apache.size | quote }}
+{{- if .Values.nginx.persistence.apache.storageClass }}
```
Comment (mgoodness) [inline stable/mediawiki/templates/apache-pvc.yaml]: Delete `nginx.`

**Review by mgoodness — CHANGES_REQUESTED** [review]

File: stable/mediawiki/templates/apache-pvc.yaml (hunk 1 above)
Comment (gyliu513) [reply stable/mediawiki/templates/apache-pvc.yaml]: Done, thanks @mgoodness

**Comment by gyliu513** [comment]
@mgoodness ping

**Pull request merged**